crates/bws/src/util.rs

```diff
@@ -0,0 +1,55 @@
+use regex::Regex;
+use uuid::Uuid;
+
+const VALID_POSIX_NAME_REGEX: &str = "^[a-zA-Z_][a-zA-Z0-9_]*$";
+
+pub(crate) fn is_valid_posix_name(input_text: &str) -> bool {
+    match Regex::new(VALID_POSIX_NAME_REGEX) {
+        Ok(r) => r.is_match(input_text),
+        Err(_) => false,
+    }
```

Comment:
I wonder if we would want to panic here instead of handling invalid regex? Unless we modify the regex we know it will be a valid regex, and the below tests executes the code which should catch regressions.

Even with a regression silently marking all values as invalid posix might be confusing and using `expect` would provide an accurate error message.

```suggestion
    Regex::new(VALID_POSIX_NAME_REGEX)
        .expect("invalid VALID_POSIX_NAME_REGEX")
        .is_match(input_text)
```

Reply:
Updated in 5bdf99c.